virttest/libvirt_xml/devices/interface.py

```diff
@@ -11,7 +11,7 @@
 
 class Interface(base.TypedDeviceBase):
 
-    __slots__ = ('source', 'mac_address', 'bandwidth',
+    __slots__ = ('source', 'hostdev_address', 'managed', 'mac_address', 'bandwidth',
```

Comment:
you'd better add the slots with same format as before, the first line is much longer than others

Reply:
Updated.